scripts/install-go.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
this is a total nit:  I think we should rename this to circle-go.sh since it both installs go and preps the circle container environment to do go more correctly.

Alternatively, we could factor out the install-go from the prep-for-circle parts.